test/js/specs/happy.js

```diff
@@ -409,4 +409,184 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
+    //   poaGuidance.verifySubtitle('Must be issued in the last 12 months')
+    //   poaGuidance.verifyFullNameText('Full name')
+    //   poaGuidance.verifyCurrentText('Current')
+    //   poaGuidance.verifyAddressText('Address')
+    //   poaGuidance.verifyIssueDateText('Issue date or')
+    //   poaGuidance.verifySummaryPeriodText('Summary period')
+    // })
+
+    it('should upload Bank Stetement and finish flow', async () => {
+      const verificationCompleteCopy = verificationComplete.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnBankIcon()
+      poaGuidance.clickOnContinueButton()
+      uploadFileAndClickConfirmButton('national_identity_card.pdf')
+      documentSelector.passportIcon.click()
+      uploadFileAndClickConfirmButton('passport.jpg')
+      uploadFileAndClickConfirmButton('face.jpeg')
+      verificationComplete.verifyUIElements(verificationCompleteCopy)
+    })
+
+    it('should upload Utility Flow and finish flow', async () => {
+      const verificationCompleteCopy = verificationComplete.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnUtilityBillIcon()
+      poaGuidance.clickOnContinueButton()
+      uploadFileAndClickConfirmButton('national_identity_card.pdf')
+      documentSelector.passportIcon.click()
+      uploadFileAndClickConfirmButton('passport.jpg')
+      uploadFileAndClickConfirmButton('face.jpeg')
+      verificationComplete.verifyUIElements(verificationCompleteCopy)
+    })
+
+    it('should upload Councit Tax Letter and finish flow', async () => {
+      const verificationCompleteCopy = verificationComplete.copy()
+      goToPoADocumentSelectionScreen()
+      poaDocumentSelection.clickOnCouncilTaxLetterIcon()
+      poaGuidance.clickOnContinueButton()
+      uploadFileAndClickConfirmButton('national_identity_card.pdf')
+      documentSelector.passportIcon.click()
+      uploadFileAndClickConfirmButton('passport.jpg')
+      uploadFileAndClickConfirmButton('face.jpeg')
+      verificationComplete.verifyUIElements(verificationCompleteCopy)
+    })
+
+    //the test below will fail because of the bug CX-3799, footer hovers the benefits letter cell
+    // it('should upload Benefits Letter and finish flow', async () => {
+    //   const verificationCompleteCopy = verificationComplete.copy()
+    //   goToPoADocumentSelectionScreen()
+    //   poaDocumentSelection.clickOnBenefitsLetterIcon()
+    //   poaGuidance.clickOnContinueButton()
+    //   uploadFileAndClickConfirmButton('national_identity_card.pdf')
+    //   documentSelector.passportIcon.click()
+    //   uploadFileAndClickConfirmButton('passport.jpg')
+    //   uploadFileAndClickConfirmButton('face.jpeg')
+    //   verificationComplete.verifyUIElements(verificationCompleteCopy)
+    // })
+
+    it('should succesfully complete cross device e2e flow with PoA document and selfie upload', async () => {
+      const mobileConnectedCopy = crossDeviceMobileConnected.copy()
```

Comment:
Didn't you test this already in the E2E cross device PR?

Reply:
Not with the PoA. This includes the upload of the PoA doc, the Cross-device sync tests, don't.

Reply:
Yes, but the cross device copy has already been tested in the e2e cross device test so I don't think you need to test it again. You just need to be able to click through the screen and be able to complete the poa flow using cross device.